### PR TITLE
Merch index sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
- pulled in Margo's branch "merchant_items_mf"
- She wrote completed tests and functionality for stories 42-44
- Add boolean active? column to the merchants table
- Create a partial for the merchant and admin merchant index pages
- Added a disable and enable method to the admin/merchant controller
- Created a deactivate_items and an activate_items method in the merchant model
- Cleaned up the merchant index views
- The user can now go to the merchant show page and view all the merchant's items minus the inactive ones
- Wrote tests for all written functionality 
- All tests are passing and are at 100% coverage for features and 99% coverage for models

### CODE REVIEW

**Comment by StephanieFriend** [comment]
-Updated the seed file with a regular user and changed the emails and passwords via the instructors request

**Comment by StephanieFriend** [comment]
- Add orders and item_orders to the seed file

File: app/models/merchant.rb [hunk 1]
```diff
@@ -30,4 +31,16 @@ def pending_orders
     Order.joins(:items).where(items: {merchant_id: self.id}).where(orders: {status: "Pending"}).distinct
   end
 
+  def deactivate_items
```
Comment (margoflewelling) [inline app/models/merchant.rb:34]: Nice, good logic. Very clean.

File: db/seeds.rb [hunk 2]
```diff
@@ -22,14 +24,44 @@
...
-                       password_confirmation: "henry2004",
+                       email_address: "merchant@example.com",
+                       password: "password_merchant",
+                       password_confirmation: "password_merchant",
                        role: 1,
                        merchant_id: bike_shop.id
-                      })
\ No newline at end of file
+                      })
+bert = User.create({name: "Bert",
```
Comment (margoflewelling) [inline db/seeds.rb:48]: Sweet, thanks for adding these seeds!

**Pull request merged**